pkg/arvo/sys/hoon.hoon

```diff
@@ -291,22 +292,25 @@
   ::  sample type of `*`.
   $-(* *)
 ::
-+*  list  [item]
+++  list
+  |$  [item]
```

Comment:
Can one avoid brackets if one has a single argument?

Reply:
Not as written, no.